### ISSUE TITLE
Non-functional browser on 2.1.0 - twice

### ISSUE THREAD

**Issue by Armastan** [Plailect/Guide#356]
Hello!

I downgraded an O3DS from 11.0 with Fieldrunners, and got to the point where I just needed to install arm9loaderhax on 2.1.0, however, the first time I got a non-functional browser, where as soon as I started it, I got an error to turn off my console. According to the guide, I had to go to 4.X and use Decryp9 to restore a NAND backup, so I used Fire Emblem Awakening to get the console to 4.4 but the browser was still freezing as soon as I opened it, why should it work on 4.x if it didn't on 2.1.0?

I realized that, since I had Fieldrunners, I should just update back to 11.0 and downgrade to 9.2.0 to use OTP Helper and get the 2.1.0 emuNAND injected back into the sysNAND and try again, so I did this and got a non-functional browser again.

Now I will try to go back where I did the sysNAND_original and emuNAND_original backups and follow the guide from there, but I would like to know why this happens and if there's a workaround so I won't get a non-functional browser on 2.1.0 for a third time.

A note should be added that if you did the DSiWare Hax, and got stuck on 2.1.0 on an O3DS, you can update to latest version and downgrade again.

EDIT:

So, I followed the new guide and got to downgrade to 2.1.0 again, but I still have a non-functional browser, the same error as before.
Here's the Dec9 log if anyone is interested: http://pastebin.com/6AgsiiGc
Thanks for the time being.

**Comment by Margen67** [comment]
Try manually installing a CIA of the browser

**Comment by Armastan** [comment]
I assume that I need version 4 on 2.1.0 emuNAND, is that correct? How could I install it?

**Comment by Armastan** [comment]
I just downgraded again to 2.1.0, going from the original backups and got a non-functional browser.

**Comment by james-d-elliott** [comment]
Attempt to force another downgrade while still on 2.1.0, it should fix any apps that have not been updated. You may also need to remove the SD card to open the browser, reinsert it again before you trigger the hax.

**Comment by Armastan** [comment]
Loading the browser with SD card inserted or not makes no difference. I should note that I can make the initial setup and see the browser for a split second before the "An error has occurred" screen.

How should I do the second downgrade?

And, is there anything useful I can load from DSiWare Hax while on 2.1.0? Since I can use it while on that version.

**Comment by james-d-elliott** [comment]
Do you still have access to SysNAND homebrew? I'm assuming that the browser downgrade failed. If you don't have access... it's possible a sys format may fix it but may just make things worse. Ideally you'd get into PlaiSysUpdater again and just do the downgrade to 2.1.0. :\

**Comment by Armastan** [comment]
I've been downgrading and updating ever since that day. I can't tell if the browser downgrade failed, since I never got any error and double-checked every downgraded emuNAND with OTP Helper.

How could a system format fix a non-functional browser? How does a update to 4.x would do either (as mentioned on the stuck on 2.1.0 page)?

I'm not really in the mood to downgrade a 6th time without any warranty, to be honest. On #3dshacks someone mentioned ctrnand flashing and that I should wait for the process on this guide or head to #Cakey (which might do during the week) to see if I could help to test the process.

That guide is linked at the start of the part 4, but I'm not really stuck on 2.1.0, since I can just update back to 11.0 and downgrade again with DSiWare Hax, I just can't install arm9loaderhax on 2.1.0 because I have no browser.

**Comment by Plailect** [comment]
New guide is up and all currently open issues are being closed. If you feel your issue still applies to the new guide, you can request for it to be re-opened by replying to it.

**Comment by Armastan** [comment]
I still get a non-functional browser on 2.1.0 after doing the ctrtransfer process.

**Comment by Armastan** [comment]
Can I get my issue re-opened? The error persists.

**Comment by Plailect** [comment]
Any progress made on this? If not, check your region settings and change them go one matching your region.

**Comment by Armastan** [comment]
Once I was in 2.1.0 someone asked if I could do a system format. I remember I used the information matching my country. 

I don't have the console right now to check, but I'll see tomorrow and post it.

**Comment by dark-samus** [comment]
Only do a system format if you're on an o3ds, not on 2ds or n3ds, as both get stuck in loops, which will brick the console... This issue sounds very odd, something is off about your console... Getting back to 4.x and using luma-dev (through MSET), enabling the errDisp patch and attempting to open the browser should provide us with some more info

**Comment by Armastan** [comment]
I don't have a flashcard for the MSET, can I use the errDisp once I update to 9.x (with a game)?

**Comment by Plailect** [comment]
Yes

**Comment by dark-samus** [comment]
Assuming the browser still doesn't work, sure. I suppose you could also
preflash the ROP while on 9.2 if the browser works, then once downgraded
back to 4.x, the ROP will already be installed (since it's stored in the
WiFi SPI flash, and not anywhere near the CTRNAND)

**Comment by Armastan** [comment]
So... I just bricked the console. Asked a friend for his Sky3ds v2 with TriForce Heroes. The game didn't show up anywhere but in the 2.1.0 3DS (DiskWritter also threw an error), since it was there and it asked me to update I decided to try. Well... Seems like I'll get a hardmod.

**Comment by Armastan** [comment]
So... I just bricked the console. Asked a friend for his Sky3ds v2 with TriForce Heroes. The game didn't show up anywhere but in the 2.1.0 3DS (DiskWritter also threw an error), since it was there and it asked me to update I decided to try. Well... Seems like I'll get a hardmod.

**Comment by efrainvs** [comment]
Armastan disculpa que te hable por aqui pero estaba usando tu guia para el 3ds podrias apoyarme por que estaba usando la guia anterior y me quede en el downgrade al 2.X.X por favor podrias apoyarme mi correo es efrain.vazquez.sosa@hotmail.com saludos gracais

**Comment by chriseyj2002** [comment]
I had a non functional browser on a Eur 2DS after downgrading 11.2 - 2.1.0. I tried resetting history and browser data, pulling battery, a game & the SD card in & out on browser e.t.c but I still couldn't get it to work.

My issue was that I as soon as I tried to type anything the keyboard would pop up and freeze and I'd have to hard reset. I could however load the Nintendo bookmarks page or click on the google or yahoo bookmarks and they would load.

In the end I used FreeProxy to setup a proxy on my laptop with a ban list that redirected yahoo or google pages to the browser hack url.

This sounds complicated but it took 10 mins with a guide. I had to enter the proxy on the 2DS in the wireless connection in system settings as I cannot type anything in the browser without a crash!

Now as soon as I click one of the bookmarks the simple proxy running on my laptop redirected the page to the browser hack one.

This worked after 2 attempts and I got back to 11.2 with custom firmware.

 Just thought I would share this in case anyone else has a similar browser issue and panics as I did.
